### ISSUE TITLE
Non-donor data

### ISSUE THREAD

**Issue by adriennefriend** [django/djangoproject.com#571]
Not sure if this is possible, but it would be helpful to have a report of non-donors who are okay with being contacted by email by the DSF. 

In addition to reaching out to past donors, I would like to write a separate message to be sent to non-donors who have agreed to be contacted by us. 

Is there anywhere on the website where we collect names and emails, and permission to contact? If not, could this be added?

**Comment by timgraham** [comment]
No, we don't collect anything on the website. Not sure if we need some bureaucracy like a privacy policy before we start doing it and I don't see much reason to since we have other mailing lists like django-users (30K members). I guess you could create a separate mailing list if you wanted to but I think we have enough of those.

**Comment by adriennefriend** [comment]
Ok, so no way to target existing non-donors, except through mailing lists.

I will add to https://github.com/django/djangoproject.com/pull/569 a note that we should send out a message to django-users on the day of the launch of the new fundraiser.

**Comment by timgraham** [comment]
We have a pretty big audience for the Django Project website too. We could enhance the "Person donated to the Django Software Foundation to support Django development. Donate today!" that appears on the sidebar of every page. Maybe add a progress bar or something? To be discussed in another issue...
